README.md

```diff
@@ -52,7 +52,29 @@ You can get the latest sources
 <!-- markdown-toc end -->
 
 # Installation
+There are multiple alternative ways to install `mastodon-archive` on your machine:
 
+## Linux Packages
+There are now packages available for Debian (`*.deb`) and Redhat (`*.rpm`) based
+systems. They are not in the standard repositories, though – but for now can be
+installed from [IzzySoft's Repositories](https://apt.izzysoft.de/). There you
+also find instructions on how to include them with your
```

Comment:
```suggestion
also find instructions on how to include them.
```

Reply:
I know this sounds fine when read aloud, but visually I think paragraphs should always end with punctuation. I guess I'm taking my new day job as tech writer a bit too serious. :laughing:

Reply:
Punctuation, hm? May we double it then?

    also find instructions on how to include them:

Reply:
Work for me.

Reply:
Done then (comes with the next push).